<h2 {% if site.style == 'dark' %} class="text-white" {% endif %}>Research interests</h2>
<p class="f4 mb-4 {% if site.style == 'dark' %}text-white{% else %}text-gray{% endif %}">I carry out research in genetic epidemiology and precision medicine with focus on autoimmune and infectious diseases.</p>
<p class="f4 mb-4 {% if site.style == 'dark' %}text-white{% else %}text-gray{% endif %}">Genetic susceptibility to a disease is typically determined by tiny effects from thousands of sites on the genome at which people differ from each other. I am developing new methods of genetic analysis, based on adding up the effects of all these variant sites on each gene to identify the core genes that cause disease.</p>

<h2 {% if site.style == 'dark' %} class="text-white"{% endif %}>Projects</h2>
<p class="f4 mb-4 {% if site.style == 'dark' %}text-white{% else %}text-gray{% endif %}">My research projects/code
  repositories.</p>
  <div class="d-sm-flex flex-wrap gutter-condensed mb-4">
    <div class="github-component d-flex flex-column flex-justify-between height-full text-left border border-gray-light bg-white rounded-1 p-3">
      <h1 class="f4 lh-condensed mb-1">Genomewide aggregation of trans- effects (GATE) method</h1>
      <p><a href="https://github.com/molepi-precmed/trans-qtls">GATE computational pipeline</a></p>
      <h2 class="f4 lh-condensed mb-1 text-gray">Description</h2>
      <p class="text-gray">
          The method is based on the Omnigenic model <a href="https://doi.org/10.1016%2Fj.cell.2017.05.038">(Boyle et al., 2017)</a> and can be applied to detect effector (core) genes mediating the disease.
          The pipeline makes use of the <a href="https://genoscores.cphs.mvm.ed.ac.uk/">GENOSCORES</a> platform to compute eQTL and pQTL scores and
          aggregate the trans QTL effects into genomewide scores which are then tested for association with the disease.
      </p>
      <p>Work is supported by <a href="https://gtr.ukri.org/projects?ref=MR%2FT032340%2F1">UK/Canada Collaboration on the genetics of long-term diabetes complications.</a></p>
      <hr>
      <div class="d-flex f6">
        <span class="repo-language-color ml-0 mr-1" style="background-color: #198CE7"></span>
        <span class="mr-3" itemprop="programmingLanguage">R</span>
        <span class="repo-language-color ml-0 mr-1" style="background-color: #3af811"></span>
        <span class="mr-3" itemprop="programmingLanguage">Shell</span>
      </div>
    </div>
  </div>

  <div class="d-sm-flex flex-wrap gutter-condensed mb-4">
    <div class="github-component d-flex flex-column flex-justify-between height-full text-left border border-gray-light bg-white rounded-1 p-3">
      <h1 class="f4 lh-condensed mb-1">GENOSCORES platform</h1>
      <p><a href="https://genoscores.cphs.mvm.ed.ac.uk">GENOSCORES website</a></p>
      <h2 class="f4 lh-condensed mb-1 text-gray">Description</h2>
      <p>I am a lead developer and maintainer of the GENOSCORES code base, server and database.</p>
      <p>GENOSCORES consists of:</p>
      <ul class="d-flex flex-column flex-justify-between text-left p-3">
          <li>curated database of published GWAS summary statistics</li>
          <li>web platform</li>
          <li>R package</li>
          <li>suite of software tools</li>
          <li>REST API</li>
      </ul>
      <p>
          GENOSCORES is used with any individual-level dataset consisting of genome-wide SNP
          genotypes to calculate genotypic scores for the intermediate phenotypes stored in the database.
      </p>
      <p>Work is supported by The Academy of Medical Sciences Springboard award</p>
    <hr>
      <div class="d-flex f6">
        <span class="repo-language-color ml-0 mr-1" style="background-color: #198CE7"></span>
        <span class="mr-3" itemprop="programmingLanguage">R</span>
        <span class="repo-language-color ml-0 mr-1" style="background-color: #4F5D95"></span>
        <span class="mr-3" itemprop="programmingLanguage">PHP</span>
        <span class="repo-language-color ml-0 mr-1" style="background-color:#e34c26"></span>
        <span class="mr-3" itemprop="programmingLanguage">HTML</span>
        <span class="repo-language-color ml-0 mr-1" style="background-color:#3af811"></span>
        <span class="mr-3" itemprop="programmingLanguage">Shell</span>
        <span class="repo-language-color ml-0 mr-1" style="background-color:#b45f06"></span>
        <span class="mr-3" itemprop="programmingLanguage">SQL</span>
      </div>
    </div>
  </div>
  
  <div class="d-sm-flex flex-wrap gutter-condensed mb-4">
    <div class="github-component d-flex flex-column flex-justify-between height-full text-left border border-gray-light bg-white rounded-1 p-3">
      <h1 class="f4 lh-condensed mb-1">Prediction of treatment response using gene expression data for genes predictive of synovial pathotypes in people with rheumatoid arthritis.</h1>
      <h2 class="f4 lh-condensed mb-1 text-gray">Description</h2>
      <p>I train predictive models with hierarchical shrinkage priors using <a href="https://github.com/mcol/hsstan">hsstan</a> package to predict treatment response using panels of biomarkers predictive of synovial pathotypes in RA.</p>
      <p>This is a collaborative project within <a href="https://gtr.ukri.org/projects?ref=MR%2FV012509%2F1">TRACT-RA</a> collaboration.</p>
    <hr>
      <div class="d-flex f6">
        <span class="repo-language-color ml-0 mr-1" style="background-color: #198CE7"></span>
        <span class="mr-3" itemprop="programmingLanguage">R</span>
      </div>
    </div>
  </div>
  
  <div class="d-sm-flex flex-wrap gutter-condensed mb-4">
    <div class="github-component d-flex flex-column flex-justify-between height-full text-left border border-gray-light bg-white rounded-1 p-3">
      <h1 class="f4 lh-condensed mb-1">Discovery of genetically predicted biomarkers mediating systemic lupus.</h1>
      <h2 class="f4 lh-condensed mb-1 text-gray">Description</h2>
      <p>I use <a href="https://genoscores.cphs.mvm.ed.ac.uk/">GENOSCORES</a> and <a href="https://github.com/molepi-precmed/trans-qtls">GATE</a> method to detect effector (core) genes mediating systemic lupus.</p>
      <p>This is a collaboration with Centro Pfizer – Universidad de Granada – Junta de Andalucía de Genómica e Investigación Oncológica <a href="https://www.genyo.es/centre/?lang=en">GENYO</a>.</p>
    <hr>
      <div class="d-flex f6">
        <span class="repo-language-color ml-0 mr-1" style="background-color: #198CE7"></span>
        <span class="mr-3" itemprop="programmingLanguage">R</span>
      </div>
    </div>
  </div>
</div>
<div class="d-sm-flex flex-wrap gutter-condensed mb-4">
  <div class="github-component d-flex flex-column flex-justify-between height-full text-left border border-gray-light bg-white rounded-1 p-3">
    <h1 class="f4 lh-condensed mb-1"><a href="https://spindlefea.soton.ac.uk">Spindle FEA</a></h1>
    I developed an application to perform parametric modelling of elastic stability of the
    mitotic spindle in several cell types during mitosis. This work was
    carried out as part of my PhD.
    <hr>
    <div class="d-flex f6">
      <span class="repo-language-color ml-0 mr-1" style="background-color:#3572A5"></span>
      <span class="mr-3" itemprop="programmingLanguage">Python</span>
      {% octicon repo height:16 class:"mr-2 v-align-middle" %}
      <a href="https://git.soton.ac.uk/ai1v14/SpindleFEA">SpindleFEA repo</a>
    </div>
  </div>
</div>

<div class="d-sm-flex flex-wrap gutter-condensed mb-4">
  <div class="github-component d-flex flex-column flex-justify-between height-full text-left border border-gray-light bg-white rounded-1 p-3">
    <h1 class="f4 lh-condensed mb-1">3D printing service</h1>
    <p>
        I co-initiated, managed and developed the web platform for the 3D printing service
        at the University of Southampton.
    </p>
    <p class="text-gray">The access is restricted only to the registered users
        since UoS moved the application behind the firewall.</p>
    <hr>
    <div class="d-flex f6">
      <span class="repo-language-color ml-0 mr-1" style="background-color: #4F5D95"></span>
      <span class="mr-3" itemprop="programmingLanguage">PHP</span>
      <span class="repo-language-color ml-0 mr-1" style="background-color:#e34c26"></span>
      <span class="mr-3" itemprop="programmingLanguage">HTML</span>
      <span class="repo-language-color ml-0 mr-1" style="background-color: #f1e05a"></span>
      <span class="mr-3" itemprop="programmingLanguage">JavaScript</span>
    </div>
  </div>
</div>

<h3 class="{% if site.style == 'dark' %}class="text-white"{% endif %}">Repositories</h3>

<div class="d-sm-flex flex-wrap gutter-condensed mb-4">
  {% if site.projects.sort_by == 'stars' %}
  {% assign sort_order = 'stargazers_count', 'last' %}
  {% else %}
  {% assign sort_order = 'pushed_at' %}
  {% endif %}

  {% if site.projects.exclude.archived && site.projects.exclude.forks %}
  {% assign filtered_repos = site.github.public_repositories | where:'archived', false | where:'fork', false | sort: sort_order | reverse %}
  {% elsif site.projects.exclude.archived %}
  {% assign filtered_repos = site.github.public_repositories | where:'archived', false | sort: sort_order | reverse %}
  {% elsif site.projects.exclude.forks %}
  {% assign filtered_repos = site.github.public_repositories | where:'fork', false | sort: sort_order | reverse %}
  {% else %}
  {% assign filtered_repos = site.github.public_repositories | sort: sort_order | reverse %}
  {% endif %}

  {% for repository in filtered_repos | limit: site.projects.limit %}
  {% unless site.projects.exclude.projects contains repository.name %}
  <div class="col-sm-6 col-md-12 col-lg-6 col-xl-4 mb-3">
    {% include repo-card.html %}
  </div>
  {% endunless %}
  {% endfor %}
</div>
